### PR TITLE
Add com.github.horaciodrs.tradesim

### PR DESCRIPTION
**Pull request by judirraquel**
Hi, my name is Horacio and I am a TradeSim developer. https://github.com/horaciodrs/tradesim

TradeSim is a Trading Simulator writen in Vala.

Thanks

### CODE REVIEW

File: com.github.horaciodrs.tradesim.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "app-id": "com.github.horaciodrs.tradesim",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "base":"io.elementary.BaseApp",
+    "base-version": "juno-19.08",
```
Comment (barthalion) [inline com.github.horaciodrs.tradesim.json:7]: This probably should be 20.08.

**Comment by barthalion** [comment]
bot, build com.github.horaciodrs.tradesim

File: com.github.horaciodrs.tradesim.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        "--talk-name=org.gtk.vfs.*"
+    ],
+    "modules": [
+        {
+            "name": "tradesim",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/horaciodrs/tradesim"
```
Comment (bilelmoussaoui) [inline com.github.horaciodrs.tradesim.json:22]: This should be a tagged release/specific commit

**Comment by judirraquel** [comment]
The build process fail checking AppStream xml
I have added this file in /Tradesim/data/com.github.horaciodrs.tradesim.xml
It is correct? if not, where should put this file?

File: com.github.horaciodrs.tradesim.json (hunk 1 above)
Comment (judirraquel) [reply com.github.horaciodrs.tradesim.json:7]: I tested this json file with flatpak-build and is working. The base-version, I think is correct. The build log says problem is on appdata.xml

Comment (bilelmoussaoui) [reply com.github.horaciodrs.tradesim.json:7]: No it's not. juno-19.08 is based on fdo 19.08 like gnome 3.36. 3.38 is based on top of fdo 20.08.

Comment (judirraquel) [reply com.github.horaciodrs.tradesim.json:7]: Ok. I changed runtime-version from 3.38 to 3.36 and also change the "com.github.horaciodrs.TradeSim.appdata.xml" filename to lower case "com.github.horaciodrs.tradesim.appdata.xml"
Please you can try run build process to send me de feedback

Comment (bilelmoussaoui) [reply com.github.horaciodrs.tradesim.json:7]: No it's not the runtime that you have to downgrade, but rather upgrade the base version.

Comment (judirraquel) [reply com.github.horaciodrs.tradesim.json]: Ok, Thank you for your fast answer. I have changed version number according to you say. Its now correct?

**Comment by bilelmoussaoui** [comment]
bot, build com.github.horaciodrs.tradesim

**Comment by judirraquel** [comment]
Is working?

**Comment by judirraquel** [comment]
Yes I checked this flatpak install --user https://dl.flathub.org/build-repo/29785/com.github.horaciodrs.tradesim.flatpakref and is working

**Comment by judirraquel** [comment]
Sorry it is my first app submition. I have do anything more?

**Pull request closed without merging**